docs/src/main/sphinx/release/release-458.md

```diff
@@ -0,0 +1,100 @@
+# Release 458 (13 Sep 2024)
+
+## General
+
+* Improve performance for queries with a redundant `DISTINCT` clause. ({issue}`23087`)
+
+## Druid connector
+
+* Reduce data transfer from remote systems for queries with large `IN` lists. ({issue}`23381`)
+
+## Delta Lake connector
+
+* {{breaking}} Deactivate [legacy file system support](file-system-legacy) for
+  all catalogs. You must activate the desired [file system
+  support](file-system-configuration) with
+  `fs.native-azure.enabled`,`fs.native-gcs.enabled`, `fs.native-s3.enabled`, or
+  `fs.hadoop.enabled` in each catalog. Use the migration guides for [Azure
+  Storage](fs-legacy-azure-migration), [Google Cloud
+  Storage](fs-legacy-gcs-migration), and [S3](fs-legacy-s3-migration) to assist
+  if you have not switched from legacy support. ({issue}`23343`)
+* Add JMX monitoring to the native S3 file system support. ({issue}`23177`)
+* Reduce the number of file system operations when reading from Delta Lake
+  tables. ({issue}`23329`)
+
+## Exasol connector
+
+* Reduce data transfer from remote systems for queries with large `IN` lists. ({issue}`23381`)
+
+## Hive connector
+
+* {{breaking}} Deactivate [legacy file system support](file-system-legacy) for
+  all catalogs. You must activate the desired [file system
+  support](file-system-configuration) with
+  `fs.native-azure.enabled`,`fs.native-gcs.enabled`, `fs.native-s3.enabled`, or
+  `fs.hadoop.enabled` in each catalog. Use the migration guides for [Azure
+  Storage](fs-legacy-azure-migration), [Google Cloud
+  Storage](fs-legacy-gcs-migration), and [S3](fs-legacy-s3-migration) to assist
+  if you have not switched from legacy support. ({issue}`23343`)
+* Add JMX monitoring to the native S3 file system support. ({issue}`23177`)
+* Reduce the number of file system operations when reading tables with file system
+  caching enabled. ({issue}`23327`)
+* Improve the `flush_metadata_cache` procedure to include flushing the file
+  status cache. ({issue}`22412`)
+* Fix failure when listing Hive tables from Glue having unsupported syntax. ({issue}`23253`)
```

Comment:
I don't understand what "unsupported syntax" means in this context. 

cc: @pajaks

Reply:
but what is a Hive unsupported table

Reply:
> but what is a Hive unsupported table

It's table which cannot be converted to Trino table and throws for example: `HIVE_UNSUPPORTED_FORMAT`